### DISCUSSION TITLE
Contributing Dropbox/Store to Mobile Native Foundation

### DISCUSSION THREAD

**Discussion by digitalbuddha** [MobileNativeFoundation/discussions#169] [Proposals]
Hi folks, in 2017 I began a project called [Store](https://github.com/nytimes/Store) under the New York Times Org while I was employed there. Store is a library that fills the gap in Android architecture - primarily data loading. A few years/jobs later The New York Times was no longer able to give the project the care it deserves and I created a new fork at my then current employer [Dropbox](https://github.com/dropbox/Store). NY Times were in support and archived their repo.  Fast forward to now, I am no longer at Dropbox and the company's investment in open source is not what it used to be.  Rather than having to move the project to yet another corporate home, I wanted to start the discussion about bringing Store into the Mobile Native Foundation. 

  The project should be open source and under a permissible license
  ✅ Store carries an Apache 2.0 license.
  
  There should be a community interest in the project
  ✅ Store has hundreds of forks and has pretty active community that helps with bugs/improvements. For example it was selected for Google's Summer of Code (which dropbox unfortunately, decided to not pursue once I left the company). We have had over 30 external contributors over the last 5 years as well.
  
  There should be a suitable scope for the community
  ✅ This project is useful for Android apps of any size/scope, it is broken up into modules allowing apps to leverage only the parts that are needed (caching, persistence, fetching)
  
  The project should be non-trivial
  ✅ Inarguably non-trivial, I have not seen a similar project/solution since this one was started.
  
  The adoption should serve a purpose
  ✅ The original contributor(s) no longer work at Dropbox, and while the project is was maintained by the Dropbox org, it has languished since I left and lost merge rights
  
  The project should be "production ready"
  ✅ The project is used by (at least) a dozen production apps, sorry hard to tell, those are just the ones I have seen bug reports from: Google, Dropbox, NY Times, Nike, Tonal as well as some prominent android sample repos like [Tivi](https://github.com/chrisbanes/tivi). 
  
  The existing maintainers should ideally want to continue in that role
  ✅  I will absolutely continue to contribute to the project but hope moving to MNF will continue to grow contributors

Dropbox Legal has agreed to let go of the project, the only step left is to figure out if MNF would like to be the new org that takes it.

Alternatively, I would create a StoreOrg/Store or something similar to stop flipping betwen corporate sponsors. Thank you kindly for the consideration/discussion

**Comment by tyvsmith** [comment]
Lgtm :shipit:

**Comment by mwolfson** [comment]
This really sounds like a great idea!
